go-controller/pkg/ovn/ovn.go

```diff
@@ -113,6 +113,9 @@ func NewOvnController(kubeClient kubernetes.Interface, wf *factory.WatchFactory)
 		loadbalancerClusterCache: make(map[string]string),
 		loadbalancerGWCache:      make(map[string]string),
 	}
+
+	oc.registerMetrics()
+	return oc
 }
```

Comment:
@squeed looks like L117 is causing all of the test failures? This NewOvnController() is called from several places.